pkg/cli/environment_test.go

```diff
@@ -50,16 +50,17 @@ func TestEnvSettings(t *testing.T) {
 		envvars map[string]string
 
 		// expected values
-		ns, kcontext  string
-		debug         bool
-		maxhistory    int
-		kubeAsUser    string
-		kubeAsGroups  []string
-		kubeCaFile    string
-		kubeInsecure  bool
-		kubeTLSServer string
-		burstLimit    int
-		qps           float32
+		ns, kcontext   string
+		debug          bool
+		maxhistory     int
+		kubeAsUser     string
+		kubeAsGroups   []string
+		kubeCaFile     string
+		kubeInsecure   bool
+		kubeTLSServer  string
+		burstLimit     int
+		qps            float32
+		lintConfigfile string
```

Comment:
`lintConfigFile` is probably more idiomatic here

Reply:
done